lib/tasks/load_us_counties.rake

```diff
@@ -0,0 +1,14 @@
+require 'csv'
+namespace :db do
+  desc "Load US Counties"
+  task :load_us_counties => :environment do
+    if County.none?
+      puts 'loading US counties.  '
+      counties = []
+      CSV.foreach("lib//assets//us_county_list.csv") do |row|
+        counties<< {name: row[0], region: row[1]}
+      end
+      County.insert_all( counties)
```

Comment:
Maybe this should be an upsert, that way if we run this over and over it won't keep adding rows.

```suggestion
      County.upsert_all( counties)
```